demisto_sdk/commands/generate_docs/generate_docs_command.md

```diff
@@ -0,0 +1,22 @@
+## generate-docs
+Generate a README file for an Integration, Script or a Playbook.
+
+**Use-Cases**
+This command is used to create a documentation file for Demisto content files.
+
+**Arguments**:
+* **-i, --input**Path of the yml file.
+* **-o, --output** The output dir to write the documentation file into, documentation file name is README.md. If not specified, will be in the yml dir.
+* **-u, --use_cases** For integration - Top use-cases. Number the steps by '*' (i.e. '\* foo. * bar.').
```

Comment:
remove use_cases, global_permissions, command_permissions, additional_info, limitations, troubleshooting,

Reply:
I think permission argument should be one of the three options (none, per command, general).
Currently it is always per command but most of the cases the permission are general or not existing.
The actual text is easier to fill in after the file generated

Reply:
so the argument will be a -p --permissions`click.Choice` in between these 3 options.
i'll also give the user an option to attach a file for the command permissions if he wants to.

Reply:
I think you can remove permissions too. It is much easier to modify the markdown file, instead of creating new file just for permissions and use the terminal